### PR TITLE
Update readme with full stack instructions

### LINKED ISSUE #10: Update README to include backend configuration
### Suggestion / Feature Request

Make README more beginner-friendly with instructions on running the mock client and server

### PR DESCRIPTION
**Pull request by qafui**
## Description
Updates the readme for a beginner audience

### Changes
* Update quickstart guide and link to mock client and server values.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,10 +13,7 @@ npm i && npm run dev
...
 Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
 
-You can start editing the page by modifying `app/page.tsx`. The page auto-updates as you edit the file.
-
-[API routes](https://nextjs.org/docs/api-routes/introduction) can be accessed on [http://localhost:3000/api/hello](http://localhost:3000/api/hello). This endpoint can be edited in `pages/api/hello.ts`.
-
-The `pages/api` directory is mapped to `/api/*`. Files in this directory are treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
-
-This project uses [`next/font`](https://nextjs.org/docs/basic-features/font-optimization) to automatically optimize and load Inter, a custom Google Font.
+## Running the Mock-Client
```
Comment (jhodapp) [inline README.md]:
```suggestion
## Running the Ambi Mock Client to generate fake sensor values
```

File: docs/sequence-front-end.png
Comment (jhodapp) [inline docs/sequence-front-end.png:1]: This looks great @qafui! One suggestion is, in the "Ambi Mock Client" block, to mention "Edge HW Client / Ambi Mock Client". My suggestion just points out that the mock client or a real hardware client "i.e. embedded Rust application and real sensor hardware" can provide the input sensor values into the system.

**Review by jhodapp — CHANGES_REQUESTED** [review]
Looks great @qafui, I have a couple of minor suggestions.

File: docs/sequence-front-end.png
Comment (qafui) [reply docs/sequence-front-end.png:1]: Makes sense. Thanks.

**Review by jhodapp — APPROVED** [review]
Looks great, thank you David.

**Review by calebbourg — APPROVED** [review]

**Pull request merged**